examples/bri-3/components/storage/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
No need to have a separate package.json in the storage component. BRI-3 will deal with migrations on app startup.